prototype/p2p/proto/p2p.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+import "google/protobuf/any.proto";
+
+package p2p;
+
+option go_package = "/types/";
+
+enum PocketTopic {
```

Comment:
Do you think this shold be scoped to P2P or available elsewhere too?

Reply:
It was initially intended only for P2P